Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -148,6 +248,7 @@ namespace AzToolsFramework
         const auto& fullPath = domOperation.GetDestinationPath();
         auto pathEntry = fullPath[pathIndex];
         AZ_Assert(pathEntry.IsIndex() || pathEntry.IsEndOfArray(), "the direct children of a row must be referenced by index");
+        const int childCount = aznumeric_cast<int>(m_domOrderedChildren.size());
```

Comment:
any strong reason why this needs to be an integer? seems like its used to index into an dequeue?

Reply:
there was a reason, but now there's not. Fixed ;)